notification-config-service-impl/src/main/java/org/hypertrace/notification/config/service/NotificationConfigServiceConstants.java

```diff
@@ -0,0 +1,7 @@
+package org.hypertrace.notification.config.service;
+
+interface NotificationConfigServiceConstants {
+  String NOTIFICATION_RULE_CONFIG_RESOURCE_NAME = "notificationRuleConfig";
```

Comment:
Maybe the namespace should be shared since we have two different resources here, but the actual resource names are internal details to the specific store and need not have exposed constants.